### ISSUE TITLE
key-spacing rule has different column if parenthesis is there

### ISSUE THREAD

**Issue by trichins** [eslint/eslint#1458]
Run eslint with the key-spacing rule against the following file:

``` js
(function() {
    'use strict';
    var tmp = {
        'a':1,
        'b':(2-1)
    };
})();
```

The two failures are:

```
  4:6  error  Missing space before value for key "a"  key-spacing
  5:7  error  Missing space before value for key "b"  key-spacing
```

Notice how the second error has a different column from the first error.  I don't think the presence of the parenthesis should change the column.  Both of them should say column 6.
